core/src/main/java/org/apache/brooklyn/core/mgmt/persist/DeserializingClassRenamesProvider.java

```diff
@@ -54,12 +55,20 @@
     }
     
     private static Map<String, String> loadDeserializingClassRenamesCache() {
-        InputStream resource = new ResourceUtils(DeserializingClassRenamesProvider.class).getResourceFromUrl(DESERIALIZING_CLASS_RENAMES_PROPERTIES_PATH);
-        if (resource != null) {
+        return ImmutableMap.<String, String>builder()
+                .putAll(loadDeserializingClassRenamesCacheFromUrl(DESERIALIZING_CLASS_RENAMES_PROPERTIES_PATH))
+                .putAll(loadDeserializingClassRenamesCacheFromUrl(KARAF_DESERIALIZING_CLASS_RENAMES_PROPERTIES_PATH))
+                .build();
+    }
+
```

Comment:
We cannot use config admin here as:

1. we are working with a static class
2. It won't work if Brooklyn is launched in classic mode

We can think of migrating this to a service once we will use Karaf only. Until then, we need to stick with this approach.